ToyBox/Classes/MainUI/PartyEditor/StatsEditor.cs

```diff
@@ -18,16 +19,353 @@
 using System.IO;
 using System.Linq;
 using ToyBox.classes.Infrastructure;
+using Unity.Collections;
 using UnityEngine;
 
 namespace ToyBox {
+
     public partial class PartyEditor {
+
         public class ToyBoxAlignmentProvider : IAlignmentShiftProvider {
+
             AlignmentShift IAlignmentShiftProvider.AlignmentShift => new() { Description = "ToyBox Party Editor".LocalizedStringInGame() };
         }
+
+        public class CharacterSkeletonReplacer {
```

Comment:
Can you move the class into a separate file?
It's a little long for a nested class; and it's needed in other files too, so moving it into its own file sounds logical.

Reply:
Agreed, that makes sense. Now we just need to figure out where exactly it should be placed. :)